agent/internal/agent.go

```diff
@@ -327,6 +327,7 @@ func (a *Agent) sender(out chan *aproto.MasterMessage) events.Publisher[containe
 			case in.StatsRecord != nil:
 				msg.ContainerStatsRecord = in.StatsRecord
 			case in.Log != nil:
+				in.Log.AgentID = &a.opts.AgentID // enrich log
```

Comment:
i think we would need to set source = agent here too. if anyone further down (e.g., container.go) wants to set a more specific source, they can and then maybe we only set it here if it is nil. but it would be nice if every log coming out of agent had at least source = agent.